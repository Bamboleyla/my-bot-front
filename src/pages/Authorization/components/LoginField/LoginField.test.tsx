import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { LoginField } from ".";

describe("LoginField", () => {
  render(
    <LoginField
      values={{ value: "Login", error: false, errorText: "" }}
      setValues={() => {}}
    />
  );
  const input = screen.getByDisplayValue("Login");
  const label = screen.getByLabelText("email");

  test("Присутствие элементов", () => {
    expect(label).toBeInTheDocument();
    expect(input).toBeInTheDocument();
  });
});
