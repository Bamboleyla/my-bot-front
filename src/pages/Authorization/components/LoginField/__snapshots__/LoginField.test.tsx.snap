// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginField Проверка срабатывания setValue при вводе значения в input 1`] = `<div />`;
